WordPress/src/main/java/org/wordpress/android/datasets/ReaderPostTable.java

```diff
@@ -624,7 +635,7 @@ public static void addOrUpdatePosts(final ReaderTag tag, ReaderPostList posts) {
                 stmtPosts.bindString(16, post.getFeaturedImage());
                 stmtPosts.bindString(17, post.getFeaturedVideo());
                 stmtPosts.bindString(18, post.getPostAvatar());
-                stmtPosts.bindLong  (19, post.timestamp);
+                stmtPosts.bindDouble(19, post.timestamp);
```

Comment:
> Wondering why this attribute is declared as a double, being that elsewhere the timestamp can be held as a long.

So, until search came along, that `timestamp` column really was a timestamp, but it's only purpose has been to determine the sort order of posts.

Search results, however, include a "score" - a double which states how relevant each post is to the submitted query - and this determines how posts are sorted. To get this to work within the existing code, I changed the `timestamp` column to a double and store the "score" in there.

And now that I look at it, I confess to not being proud of it :)

What really should happen is for me to rename that `timestamp` column (and related field in the `ReaderPost` model) to better reflect its purpose. If that would address your (perfectly valid) complaint, I'd like to file that as a separate issue to avoid further bloating this PR.